resources/metadata/sonar.yaml

```diff
@@ -41,6 +41,11 @@ spec:
           - PARAMETERS
         secret: true
         resourceRef:
+          - type: vaultSecret
+            paths:
+            - $(vaultPath)/sonar
+            - $(vaultBasePath)/$(vaultPipelineName)/sonar
```

Comment:
What is the difference between `vaultBasePath` and `vaultPath`?

Reply:
I had a few sessions with @lndrschlz and we noticed that the previous lookup paths were tightly coupled with the structure that the HyperSpace portal uses. Which forces you to create the same structure in vault as the portal would do. 

So if you are not using the portal it is much simpler to just specify `vaultPath` instead of `vaultBasePath` and `vaultPipelineName`. During the vault lookup we only try paths that we can interpolate.